# Generated by Django 3.0.6 on 2020-07-21 13:52

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('exception', '0004_auto_20200721_1314'),
    ]

    operations = [
        migrations.AddField(
            model_name='validate_exception',
            name='risk_id',
            field=models.CharField(max_length=15, null=True),
        ),
    ]
